AdminSelfUpgrade.php

```diff
@@ -2073,608 +2150,570 @@ public function doUpgrade()
 			if (!file_exists(INSTALL_PATH.DIRECTORY_SEPARATOR.$upgrade_dir_php))
 			{
 				$this->next = 'error';
-				$this->next_desc = $this->l('/install/upgrade/php directory is missing in archive or directory');
-				$this->nextQuickInfo[] = '/install/upgrade/php directory is missing in archive or directory';
-				$this->nextErrors[] = '/install/upgrade/php directory is missing in archive or directory.';
+				$this->next_desc = INSTALL_PATH.$this->l(' directory is missing in archive or directory');
```

Comment:
Maybe: 
`$this->l(sprintf('%s directory is missing in archive or directory', INSTALL_PATH));`